Backend/app/therapy_sessions/serializers.py

```
@@ -0,0 +1,744 @@
+    def get_websocket_url(self, obj):
+        """Generate secure WebSocket URL for the session"""
+        if obj.is_online and obj.status in ['UPCOMING', 'IN_PROGRESS']:
+            # Use WSS (secure WebSocket) protocol
+            request = self.context.get('request')
+            if request:
+                host = request.get_host()
+                # Replace http/https with wss for secure WebSocket
+                if request.is_secure():
+                    protocol = 'wss'
+                else:
+                    protocol = 'ws'  # Use ws for development, wss for production
```

Ruff (0.12.2)

Use ternary operator `protocol = 'wss' if request.is_secure() else 'ws'` instead of `if`-`else`-block

(SIM108)